### PR TITLE
Translated 404 page to German

### PR DESCRIPTION
**Pull request by flechnical**
I followed the guide lines from here: https://github.com/qmk/qmk.fm/blob/gh-pages/readme.md
The folder for Germany should be "de-de", however I added my translation to "de" for easier merging.

### CODE REVIEW

File: _i18n/de/404.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Fehler 404 - Seite nicht gefunden"
+permalink: /de-de/404.html
+lang: "de-de"
```
Comment (kuchosauronad0) [inline _i18n/de/404.md]:
```suggestion
lang: "de"
```

File: _i18n/de/404.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Fehler 404 - Seite nicht gefunden"
+permalink: /de-de/404.html
```
Comment (kuchosauronad0) [inline _i18n/de/404.md]:
```suggestion
permalink: /de/404.html
```

File: _i18n/de/404.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+lang: "de-de"
+---
+
+Sorry, diese Seite ist möglicherweise umgezogen. Bitte kontaktiere <a href="mailto:jack@qmk.fm">jack@qmk.fm</a> mit dem Link/Inhalt, den Du gesucht hättest.
```
Comment (kuchosauronad0) [inline _i18n/de/404.md]:
```suggestion
Entschuldigung, diese Seite ist möglicherweise umgezogen. Bitte kontaktiere <a href="mailto:jack@qmk.fm">jack@qmk.fm</a> mit dem Link/Inhalt, den Du gesucht hättest.
```

**Review by kuchosauronad0 — COMMENTED** [review]
I used the ISO code from the [link](https://www.andiamo.co.uk/resources/iso-language-codes/) the readme.md provides which suggests 'de' as the format. Therefore the directory should be 'de' as well. From my understanding 'de' will act as a fallback no matter if the device is set to de-DE, de-CH or de-AU. As for the actual difference between the four it mostly comes down to the format of dates if ISO 8601 is not used as far as I am concerned. Cheers

**Review by kuchosauronad0 — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**